spec/rails_helper.rb

```diff
@@ -7,6 +7,8 @@
 require 'capybara/email/rspec'
 require 'cancan/matchers'
 require 'sidekiq/testing'
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.